src/components/link.js

```diff
@@ -0,0 +1,39 @@
+var registerComponent = require('../core/component').registerComponent;
+
+module.exports.Component = registerComponent('link', {
+  schema: {
+    url: { default: '' },
```

Comment:
or, rather can we call the component `href`? I know it conflicts with the [`aframe-href-component`](https://github.com/gasolin/aframe-href-component). another approach I thought about, in keeping with the behaviour of 2D/classic anchors, perhaps we could actually wrap the entities in `<a>` elements. any emitted `click` would bubble up to the `<a>` and handle the page navigation (i.e., modifying `window.location` from JS). thoughts?